nile-backend/src/main/java/com/tp/Nile/controllers/ReviewController.java

```diff
@@ -1,16 +1,87 @@
 package com.tp.Nile.controllers;
 
+import com.tp.Nile.exceptions.*;
 import com.tp.Nile.services.ReviewService;
+import com.tp.Nile.services.ReviewServiceImpl;
+import com.tp.Nile.models.Review;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.CrossOrigin;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
 
 @RequestMapping("/api")
```

Comment:
@zmiles17  Made a similar point on #85 that we should keep our endpoints consistent for each controller. So `@RequestMapping` should take the form "/api/{modelName}" and then each additional endpoint should be like `@GetMapping("/all")` or `@GetMapping("/{reviewId}")` or `@PostMapping("/add")`